src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -1213,6 +1215,57 @@ public void readonly() {
     sendCommand(READONLY);
   }
 
+  public void geoadd(byte[] key, double longitude, double latitude, byte[] member) {
+    sendCommand(GEOADD, key, toByteArray(longitude), toByteArray(latitude), member);
+  }
+
+  public void geoadd(byte[] key, Map<byte[], GeoCoordinate> memberCoordinateMap) {
```

Comment:
@marcosnils 
1. GeoCoordinate is used by geopos and GeoRadiusResponse. 
It represents only coordinate (point) itself, not including member. So if we change GeoCoordinate to include member, we need another class to cover coordinate.

2. Let's think about GeoCoordinate itself.
Assume we have two GeoCoordinate objects which members are same but coordinates are different. Are two objects same, and it is natural to understand without explanation?

3. It would be minor, but in real world, relation of place name and coordinate is not permanent.